alma/alma-8.x/alma-8.7-hpc/install.sh

```diff
@@ -4,6 +4,8 @@ set -ex
 # install pre-requisites
 ./install_prerequisites.sh
 
+export GPU="NVIDIA"
```

Comment:
I'd like to add the param to the other install scripts as well. Break the interface once and make it consistent.

If we want to only support AMD on U22 for now, that's fine. But we should take in the param, do validation similar to what we add to U22, but return an error message that "AMD GPUs are not supported on this distro at this time" or something similar.

Reply:
You can pass an argument to these install scripts even though it isn't read if that makes the pipeline simpler to implement. But you are saying you want it to fail loudly if someone tries to generate an AMD image for Ubuntu 20 and Alma?

Reply:
I've added checks so that alma and ubuntu-20 install scripts will fail if anything other than NVIDIA is passed to them, but by default they will build the NVIDIA image if nothing is passed in.